docs/modules.md

```diff
@@ -13,6 +13,28 @@ Bookshelf is designed with modularity in mind. Features that serve a common purp
 ```{include} _templates/featured-modules.md
 ```
 
+## 📚 Glossary
+
+Here are some of the key terms used throughout the documentation.
+
+:::{list-table}
+*   - **`ata` functions**
+    - `ata` is for "as to at". These functions takes two different positions as arguments: the one is given through the positional argument (`positioned`, `at`, _etc._), and the other one is given is the executor's position.
+*   - **Bundle**
+    - A collection of modules that are shipped together.
+*   - **Feature**
+    - Something designed for the final user in order to allow them to perform a task. Most of the time, a feature is a function tag to call instead of the function. A feature can also be a loot table, a structure, a predicate or whatever.
+*   - **Module**
+    - A group of features that share a common purpose and that are contained in a namespace (i.e. datapack structures).
+*   - **`with` function macro argument**
+    - Used to specify optional arguments.
+*   - **`run` function macro argument**
+    - Used to specify a command that should be executed by the feature, often immediately.
+*   - **`on_X` function macro argument**
```

Comment:
maybe use the same convention as for the contribution page `on_<event>`? I don't have a preference for one or the other but I think we should use the same everywhere